features/step_definitions/assertion_steps.rb

```diff
@@ -23,6 +22,11 @@
   end
 end
 
+And(/^"([^"]*)" should be set in "([^"]*)"$/) do |status, list|
+  dropdown = page.find("##{list}")
+  selected_option = dropdown.find('option[selected]').text
+  expect(selected_option).to eql status
+end
```

Comment:
Well. if you perform an action in a step that implies in the title that it is an assertion, that will always fise a red flag with me. I would like to get a second opinion on this though.